org.lflang/src/org/lflang/generator/ts/TSGenerator.kt

```diff
@@ -181,11 +181,13 @@ class TSGenerator(
             fsa.generateFile(fileConfig.srcGenBasePath.relativize(tsFilePath).toString(),
                 tsCode.toString())
             
-            if (targetConfig.dockerOptions != null) {
+            if (targetConfig.dockerOptions != null && !isFederated) {
```

Comment:
If it is federated, I think an error message should be thrown that this is unsupported (if I understand the intention here correctly).

Reply:
you're correct